tests/MongoDB.Driver.Tests/Specifications/change-streams/ChangeStreamTestRunner.cs

```diff
@@ -348,7 +347,7 @@ private void AssertSuccess(List<ChangeStreamDocument<BsonDocument>> actualResult
 
         private void AssertChangeStreamDocument(ChangeStreamDocument<BsonDocument> actualDocument, BsonDocument expectedDocument)
         {
-            JsonDrivenHelper.EnsureAllFieldsAreValid(expectedDocument, "_id", "documentKey", "operationType", "ns", "fullDocument");
+            JsonDrivenHelper.EnsureAllFieldsAreValid(expectedDocument, "_id", "documentKey", "operationType", "ns", "fullDocument", "updateDescription", "to");
```

Comment:
Only "to" needs to be added here, not "updateDescription".

Reply:
No, `updateDescription` is also required because of this result node.

https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/39/files#diff-6d800193f26bb30deae9a560ca2b3a8cR542